Code/GraphMol/SmilesParse/SmilesParseOps.cpp

```diff
@@ -208,7 +208,18 @@ typedef std::pair<int, int> INT_PAIR;
 bool operator<(const INT_PAIR &p1, const INT_PAIR &p2) {
   return p1.first < p2.first;
 }
-
+namespace {
```

Comment:
Perhaps move into QueryOps.h?  Otherwise I'm happy with this.

```
inline bool isUnsaturated(Atom const *at) {
 const ROMol *mol = at.getOwningMol(); 
 ROMol::OEDGE_ITER beg, end;
   boost::tie(beg, end) = mol->getAtomBonds(atom);
   while (beg != end) {
     const BOND_SPTR bond = (*mol)[*beg];
     ++beg;
     if (bond->getBondType() != Bond::SINGLE) return true;
   }
   return false;
 }
  return at->getHybridization();
};
```